src/start.js

```diff
@@ -412,42 +412,75 @@ ipcMain.on("set-preferences", (event, arg) => {
   currentUserSettings.set("userPreferences", arg);
 });
 
-//soemthjing else
-ipcMain.on("save-preferences", (event, arg) => {
-  const posturePref = getSetting("posture");
-  const movePref = getSetting("movement");
-  const visionPref = getSetting("vision");
-  const hydrationPref = getSetting("hydration");
-  const mindfulPref = getSetting("mindfulness");
+//update timer for posture
+ipcMain.on("updateTimer1", (event) => {
   const now = new Date().getTime();
-
-  pstTime.trigger = now + posturePref.frequency;
+  const posturePref = getSetting("posture");
+  if (!posturePref.active) {
+    pstTime.trigger = now - 60000
+  } else {
+    pstTime.trigger = now + posturePref.frequency;
+  }
   pstTime.frequency = posturePref.frequency;
   pstTime.duration = posturePref.duration;
   pstTime.active = posturePref.active;
+})
 
-  moveTime.trigger = now + movePref.frequency;
+//update timer for movement
+ipcMain.on("updateTimer2", (event) => {
```

Comment:
would it be a good idea to name the events updateMoveTime, so it's clear what activity is associated with it? (same for the others below)